cfme/base/__init__.py

```diff
@@ -65,25 +85,37 @@ def collect_logs(self):
 
     @property
     def zone(self):
-        server_res = self.appliance.rest_api.collections.servers.find_by(id=self.sid)
-        server, = server_res
+        server, = self.appliance.rest_api.collections.servers.find_by(id=self.sid)
```

Comment:
why do we have `,` after `server` ? wouldn't it throw ValueError if there are multiple values?

Reply:
yes, which is the current behavior, and intended. If you have mutliple server entities with the same ID, that's a problem.

Reply:
Can you comment it there maybe?